lib/python/treadmill_aws/cli/admin/aws/image.py

```diff
@@ -117,13 +117,13 @@ def configure(account, image):
     )
     @click.option(
         '--key',
-        required=True,
+        required=False,
```

Comment:
I'd remove this line; default is False.

Reply:
removed

Reply:
Can you remove the other required=False flags from the file?